packages/amplify-e2e-core/src/utils/rds.ts

```diff
@@ -26,12 +28,12 @@ const DEFAULT_SECURITY_GROUP = 'default';
 const IPIFY_URL = 'https://api.ipify.org/';
 const AWSCHECKIP_URL = 'https://checkip.amazonaws.com/';
 
-type RDSConfig = {
+export type RDSConfig = {
```

Comment:
There's one extra change here.
I added `export` keyword here to make this available to callers.
(exported functions below use this to type inputs, it was forgotten export that surfaced during refactor of construct tests)